app/models/clickhouse/events_enriched.rb

```diff
@@ -10,15 +10,15 @@ class EventsEnriched < BaseRecord
 #
 # Table name: events_enriched
 #
-#  code                       :string           not null, primary key
```

Comment:
Not sure these migrations should be part of this PR ;)

Reply:
damit I did it again!